examples/patterns/image/shadowed.html

```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Image / with shadow
+category: _patterns
+---
+
+<img class="p-image--shadowed" src="http://via.placeholder.com/350x350?text=*" alt="">
```

Comment:
Instead of using an asterisk (*) as text could we update to use the word 'Vanilla'

Reply:
I wanted to remove text entirely but couldn't - the reason for this is that these text is liable to minute changes (which are outside of our control) on a  regular basis which can knock off Percy

Reply:
Ok no worries let's leave it as is then